src/_guides/language/effective-dart/design.md

```diff
@@ -8,8 +8,6 @@ prevpage:
 <?code-excerpt replace="/([A-Z]\w*)\d\b/$1/g"?>
 <?code-excerpt plaster="none"?>
 
-{% include effective-dart-banner.html %}
```

Comment:
`effective-dart-banner.html` should probably be deleted since no more uses remain.